app/javascript/src/App.jsx

```diff
@@ -4,6 +4,7 @@ import { AuthProvider } from "contexts/auth";
 import { UserProvider } from "contexts/user";
 
 import Main from "./components/Main";
+import "./lib/dayjs"; // eslint-disable-line
```

Comment:
Use alias.

Reply:
Ok, I will add it to the `webpacker alias`.